tests/api_tests/nodes/test_views.py

```diff
@@ -1349,3 +1348,66 @@ def test_return_deleted_private_node_pointer(self):
         #check that deleted pointer can not be returned
         res = self.app.get(self.private_url, auth=self.basic_auth, expect_errors=True)
         assert_equal(res.status_code, 404)
+
+
+class TestReturnDeletedNode(ApiTestCase):
+    def setUp(self):
+        super(TestReturnDeletedNode, self).setUp()
+        self.user = UserFactory.build()
+        self.user.set_password('justapoorboy')
+        self.user.save()
+        self.basic_auth = (self.user.username, 'justapoorboy')
+
+        self.non_contrib = UserFactory.build()
+        self.non_contrib.set_password('justapoorboy')
+        self.non_contrib.save()
+        self.basic_non_contrib_auth = (self.non_contrib.username, 'justapoorboy')
```

Comment:
If you use the `AuthUserFactory` , you don't need to set the user's password or have this auth variable.

I've seen this in a few API PRs now; @brianjgeiger please advise devs to use `AuthUserFactory` from now on.